.travis/docker-build.sh

```diff
@@ -9,29 +9,32 @@ cd /liqd/aula
 QC_MAX_SUCCESS="100"
 QC_MAX_SIZE="30"
 
-while [[ $# > 1 ]]
-do
-key=$1
-case $key in
-    --qc-max-success)
-    QC_MAX_SUCCESS="$2"
+while [[ $# > 1 ]]; do
+    key=$1
+    case $key in
+        --qc-max-success)
+            QC_MAX_SUCCESS="$2"
+            ;;
+
+        --qc-max-size)
+            QC_MAX_SIZE="$2"
+            ;;
+
+        *)
+            ;;
+    esac
     shift
-    ;;
-
-    --qc-max-size)
-    QC_MAX_SIZE="$2"
-    shift
```

Comment:
The double shift is necessary, to get rid of the parameter and its value in the same round.